docs/source/reference/cloud-permissions.rst

```diff
@@ -0,0 +1,195 @@
+.. _cloud-permissions:
+
+Cloud Permissions Setup
+=======================
+
+SkyPilot will work out of the box for users who own the root (admin) account of their clouds, if they follow the steps in :ref:`Installation section <cloud-account-setup>`.
+
+However, in some cases, the user may want to **limit the permissions** of the account used by SkyPilot. This is especially true if the user is using a shared cloud account. The following sections describe the permissions required by SkyPilot for each cloud provider.
+
+.. _cloud-permissions-aws:
+
+AWS
+---
+
+.. _cloud-permissions-aws-user-creation:
+
+User Creation
+~~~~~~~~~~~~~
+
+AWS accounts can be attached with a policy that limits the permissions of the account. Follow these steps to create an AWS user with the minimum permissions required by SkyPilot:
+
+1. Open the IAM dashboard in the AWS console and click on the "Users" tab. Then, click "Add users" and enter the user's name.
```

Comment:
For the whole doc: for referring to buttons that appear in cloud consoles, e.g., `"Users"`, can we bold it `**Users**`? Following a common doc styleguide.